alerting/provider/custom/custom.go

```diff
@@ -79,7 +81,7 @@ func (provider *AlertProvider) buildHTTPRequest(ep *endpoint.Endpoint, alert *al
 }
 
 func (provider *AlertProvider) Send(ep *endpoint.Endpoint, alert *alert.Alert, result *endpoint.Result, resolved bool) error {
-	request := provider.buildHTTPRequest(ep, alert, resolved)
+	request := provider.buildHTTPRequest(ep, alert, resolved, result)
```

Comment:
```suggestion
	request := provider.buildHTTPRequest(ep, alert, result, resolved)
```
Same signature as Send, for consistency.